packages/bot-web-ui/src/stores/save-modal-store.ts

```diff
@@ -137,6 +102,49 @@ export default class SaveModalStore implements ISaveModalStore {
         this.toggleSaveModal();
     }
 
+    addStrategyToWorkspace = async (
```

Comment:
@vinu-deriv could you please check that no warning regarding that the call of function earlier then definition? usually definition should be earlier, this is an async function, but still...

Reply:
I can't see any warning, but I will take it above